components/Stack/Stack.tsx

```diff
@@ -0,0 +1,30 @@
+import { useRouter } from 'next/router';
+import NamedList from '../NamedList/NamedList';
+
+const STACK_TITLE = 'Стек технологий';
+const STACK_LANGUAGE = {
+  title: 'Языки и фреймворки',
+  data: ['JavaScript', 'React', 'SCSS', 'HTML', 'TypeScript', 'Electron', 'NextJS'],
+};
+const STACK_LIBRARY = {
+  title: 'Библиотеки и инструменты',
+  data: ['MobX', 'Create React App', 'BEM', 'Redux Toolkit', 'Storyblok', 'Storybook'],
+};
```

Comment:
it seems better to use a different structure for the data 
`[{title: 'Языки', data: ['JavaScript']}, {title: 'Библиотеки', data: ['MobX']}]`
this will make using the translation service more convenient